tests/all/unit/api/validation/test_tickets.py

```diff
@@ -113,6 +113,35 @@ def test_quantity_quantity_gt_min(self):
         with self.assertRaises(UnprocessableEntityError):
             TicketSchema.validate_quantity(schema, data)
 
+    def test_price_pass(self):
+        """
+        Tickets Validate Price - Tests if no exception is raised
+        :return:
+        """
+        schema = TicketSchema()
+        data = {'type': 'free'}
+        TicketSchema.validate_price(schema, data)
+
+    def test_price_lt_zero(self):
+        """
+        Tickets Validate Price - Tests if exception is raised
+        :return:
+        """
+        schema = TicketSchema()
+        data = {'type': 'paid'}
+        with self.assertRaises(UnprocessableEntityError):
+            TicketSchema.validate_price(schema, data)
+
+    def test_discount_code_pass(self):
+        """
+        Tickets Validate Price - Tests if exception is raised
+        :return:
+        """
+        schema = TicketSchema()
+        data = {}
+        original_data = {}
+        TicketSchema.validate_discount_code(schema, data, original_data)
```

Comment:
exception case is missing here

Reply:
tried, but not sure if unit test can involve queries.